src/transport/assembly_dg.hh

```diff
@@ -995,23 +995,23 @@ public:
  * Auxiliary container class sets the initial condition.
  */
 template <unsigned int dim, class Model>
-class InitConditionAssemblyDG : public AssemblyBase<dim>
+class InitProjectionAssemblyDG : public AssemblyBase<dim>
 {
 public:
     typedef typename TransportDG<Model>::EqFields EqFields;
     typedef typename TransportDG<Model>::EqData EqData;
 
-    static constexpr const char * name() { return "InitConditionAssemblyDG"; }
+    static constexpr const char * name() { return "InitProjectionAssemblyDG"; }
 
     /// Constructor.
-    InitConditionAssemblyDG(EqFields *eq_fields, EqData *eq_data)
+    InitProjectionAssemblyDG(EqFields *eq_fields, EqData *eq_data)
     : AssemblyBase<dim>(eq_data->dg_order), eq_fields_(eq_fields), eq_data_(eq_data) {
```

Comment:
@dflanderka Please, remove the order parameter of AssemblyBase as well as creation of default quadrature objects. Move this to the particular assembly classes, take order from eq_data. Some assembly classes doesn't need the harder at all, some need only face/side quadrature...